### DISCUSSION TITLE
thank you

### DISCUSSION THREAD

**Discussion by wildcat37** [ekumanov/cybrid#3] [General]
this is brilliant. thank you for publishing this, I think I might have just found an economical way to buy myself a piano with a real action.

**Comment by ekumanov** [comment]
Glad you like it although it's in a very rough shape here. There's another more advanced project inspired by this one and I've linked to it on the main page too.

**Reply by wildcat37** [reply]
Why do you say it is in a very rough shape? 

I know nothing about playing the piano, but I'm trying to acquire a midi controller with a good action so that I can learn piano and this seems to be an excellent idea, assuming I can get a low cost key bed.

Would you say the Cybrid functions as well as a kawai vpc1 midi controller or a kawai mp11 or mp11se? (if i'm not mistaken, these are the kawai models that have the grand feel actions that most resemble an horizontal acoustic piano.) I am wondering if I should build a Cybrid or buy a vpc1.

**Reply by davidedelvento** [reply]
This and Jay's controller are like the Yamaha N1 or Kawai NV10, very different and way better than VPC1 or MP11.
However your question sounds like: I want to learn to ride a bicycle, should I buy one in the shop or should I build one? I would say buy one in the shop, or even better a second hand Privia, which can be gotten for "peanuts". As a complete beginner, these differences would not matter to you for a year or two. At the same time, if you like, you can embark in this project, which will take you a year or two of work (admittedly you could be much faster than that, but only if you really have plenty of spare time, which in my opinion is better spent if you can do lots of playing/practice too). So this project will be ready when you are.

If you are in the USA, you can get a real grand action for just "peanuts", or even free, depending on where you live and how much patience you have. However you will need to study either (or better, BOTH) the Reblitz and Ingrec piano tuning and regulation books (you can obviously skip the tuning, but not the regulation).

**Reply by wildcat37** [reply]
davidedelvento, excellent information. "way better than VPC1 or MP11" is the piece of information that I wanted to know (ideally from a piano player qualified to make such a statement). and thanks for taking the time to walk through the scenario that might be best suited for a beginner. I think what you wrote is correct. There is indeed a free yamaha baby grand being offered on craigslist. No space and too loud for neighbors, so I can't have it. I can definitely buy a used Privia or new VPC1 and call it a day and focus on learning, very true. But at the same time the diy controller seems to be excellent value and superior over the mp11 if I can get a cheap key bed since I can handle the electronics/programming and probably some regulation from referencing youtube videos, but nothing by ear since I am a beginner.

**Reply by davidedelvento** [reply]
You are most welcome. Re: the Yamaha baby grand. Usually those are scams (the really free pianos tend to be the ones from obscure brands of more than 100y of age), but if it's not, you can take it, remove and keep the action, keyslip, keyboard cover (and possibly part of the case as I described IIRC [here](http://forum.pianoworld.com/ubbthreads.php/topics/2998553/re-cybergenes-diy-hybrid-midi-controller-is-finally-here.html)) and pay the disposal fee for the rest of the instrument (which will be around $300, all included). Similar to what I've done myself (but meanwhile I have two other pianos to practice on, until I finish working on this project). Best of luck and stay in touch here or at the https://github.com/jkominek/piano-conversion/

**Reply by wildcat37** [reply]
thanks for the links, didn't know about the forum posts. could be a scam, I've forgotten that craigslist is full of scammers.

**Reply by SteveMcMake** [reply]
Hello, 

I am totally new here in Gitup - and I am curios, if someone is still reacting, since this project is a kind of old. 

First of all - congratulation and thank you for sharing. 
You mentioned another project, inspired by your own, but better, more advanced. 

I did not find it. 

Where can I find it?

I would be interested to see it. I want to transfer it from a grand piano to an upright piano. I hope it will work. 

all the best to you 

Steve

**Reply by ekumanov** [reply]
https://github.com/gzweigle/open-hybrid-piano

**Comment by SteveMcMake** [comment]
Thanks a lot for this information.
Steve

Am Di., 21. Apr. 2026 um 23:51 Uhr schrieb ekumanov <
***@***.***>:
